tests/cases/fourslash/completionsLiteralOverload.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path="fourslash.ts" />
+
+// @allowJs: true
+
+// @Filename: /a.tsx
+//// interface Events {
+////   "": any;
+////   drag: any;
+////   dragenter: any;
+//// }
+//// declare function addListener<K extends keyof Events>(type: K, listener: (ev: Events[K]) => any): void;
+////
+//// declare function ListenerComponent<K extends keyof Events>(props: { type: K, onWhatever: (ev: Events[K]) => void }): JSX.Element;
+////
+//// addListener("/*ts*/");
+//// (<ListenerComponent type="/*tsx*/" />);
+
+// @Filename: /b.js
+//// addListener("/*js*/");
+
+verify.completions({ marker: ["ts", "tsx", "js"], exact: ["", "drag", "dragenter"] });
+edit.insert("drag");
+verify.completions({ marker: ["ts", "tsx", "js"], exact: ["", "drag", "dragenter"] });
```

Comment:
Should we also have a test where `drag` is already entered and we check that we don't output `""`? I mean, I guess we should just so we know what our current behavior is, I suppose.

Reply:
(The test you’re suggesting is already what’s happening on this line)

Reply:
Ahh, ok, I misread - thanks!